src/usps_processor/parse.clj

```diff
@@ -1,15 +1,53 @@
 (ns usps-processor.parse
   (:require [clojure.string :as s]
             [clojure.data.csv :as csv]
-            [turbovote.imbarcode :as imb]))
+            [clojure.tools.logging :as log]
+            [turbovote.imbarcode :as imb]
+            [clj-time.core :as time]
+            [clj-time.format :as format]
+            [clj-time.coerce :as coerce]
+            [usps-processor.zip-lookup :as zip]))
+
+(def default-timezone-id "America/Los_Angeles")
+(def usps-time-format-pattern "M/d/yyyy H:mm:ss")
+
+(defn zip->timezone-id
+  "Looks up the id of the timezone for the provided zip."
+  [zip]
+  (or (some-> zip
+              (Integer/parseInt)
```

Comment:
In a very real way, zip codes are not integers, they are strings of numbers. Treating them as numbers seems like a way to get into trouble.